pkg/errdefs/errors.go

```diff
@@ -9,20 +9,27 @@ package errdefs
 import (
 	stderrors "errors"
 	"net"
+	"strings"
 	"syscall"
 
 	"github.com/pkg/errors"
 )
 
 var (
 	ErrAlreadyExists = errors.New("already exists")
+	SignalKilled     = "signal: killed"
```

Comment:
`const signalKilled = "signal: killed"` should be better.

Reply:
fixed